src/assets/src/models.ts

```diff
@@ -69,6 +69,7 @@ export interface QueueFull extends QueueBase {
     description: string;
     hosts: User[];
     allowed_backends: string[];
+    physLocation: string;
```

Comment:
Please use `phys_location`. You don't have to do this everywhere in the FE, only where you're using the data that's coming from the backend (in contrast to the data you're handling in the form). I've tried to suggest a change where tweaking is needed and leave alone where it's not.
```suggestion
    phys_location: string;
```